### PR TITLE
Refactor of Slovak Locale For Dehumanize Support

### PR DESCRIPTION
**Pull request by anishnya**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪  Added **tests** for changed code.
- [ ] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [ ] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚  Updated **documentation** for changed code.
- [ ] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes

Refactored Slovak locale to support dehumanize. Similar to Czech locale refactor #1021.

### CODE REVIEW

File: arrow/locales.py [hunk 1]
```diff
@@ -3219,20 +3245,19 @@ def _format_timeframe(self, timeframe: TimeFrameLiteral, delta: int) -> str:
...
+        elif delta < 0:
             key = "past"
-        form: Union[List[str], str] = form[key]
-
-        if isinstance(form, list):
-            if 2 <= abs_delta % 10 <= 4 and (
+        else:
+            if "future-singular" not in form:
+                key = "future"
+            elif 2 <= abs_delta % 10 <= 4 and (
```
Comment (systemcatch) [inline arrow/locales.py:3253]: Can you just explain this part?

Comment (anishnya) [reply arrow/locales.py:3253]: Some timeframe units contain two future tenses, future-singular and future-plural. Timeframe units without these two forms just have the form "future". This check is with regards to that.

**Review by systemcatch — APPROVED** [review]
Looks good to me, feel free to merge or wait for @krisfremen to review.

**Review by krisfremen — APPROVED** [review]
Looks good to me.

**Pull request merged**